apps/api/src/deployment/services/deployment/deployment.service.ts

```diff
@@ -193,4 +194,33 @@ export class DeploymentService {
       });
     }
   }
+
+  public async list(
+    owner: string,
+    { skip, limit }: { skip?: number; limit?: number }
+  ): Promise<{ deployments: GetDeploymentResponse["data"][]; total: number; hasMore: boolean }> {
+    const pagination = typeof skip === "number" && typeof limit === "number" ? { offset: skip, limit } : undefined;
+    const deploymentReponse = await this.deploymentHttpService.loadAllDeployments(owner, "active", pagination);
+    const deployments = deploymentReponse.deployments;
+    const total = parseInt(deploymentReponse.pagination.total);
```

Comment:
I think we need to add this rule after all https://eslint.org/docs/latest/rules/radix

Reply:
To enforce the radix?